iOS-DOCS.md

```diff
@@ -0,0 +1,393 @@
+![true\[X\] logo](media/truex.png)
+
+# TruexAdRenderer tvOS Documentation
```

Comment:
iOS

Reply:
huh.. how is this possible.. I thought I replaced all those... thanks for catching